core/src/main/resources/common/message/TransactionLogValue.json

```diff
@@ -13,27 +13,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Version 1 is the first flexible version.
+// KIP-915: bumping the version will no longer make this record backward compatible.
```

Comment:
nit: I would be better to move those to right before `validVersions`.

Reply:
We could perhaps add more context about the comment as well. For instance, we should suggest to always add tagged fields from now on, etc.